workshop_materials.md

```diff
@@ -0,0 +1,104 @@
+# Workshop Materials
+
+This page is meant to act as a collection of the latest Imaging Applications team workshop materials.
```

Comment:
Maybe should add a sentence about how to find upcoming workshops -- link to RIT calendar?

Reply:
I'm not sure if a link to the RIT calendar in this public facing repo would be useful.
Probably a link to the calendar should be wherever `this` page is linked to.

Reply:
Hmm. I was thinking of putting this markdown in Sharepoint, but yeah then the link here in GitHub would block people. I think that's Ok?